app-backup/buttersink/buttersink-9999.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="rsync-like utility for btrfs snapshots"
+HOMEPAGE="https://github.com/AmesCornish/buttersink"
+
+LICENSE="GPL-3"
+SLOT=0
+
+# local tests would require root and cause sandbox issues with btrfs subvolume
+# operations, and network tests would require an SSH server with root login to
+# test the SSH backend, or remote S3 for that backend
+RESTRICT="test"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
```

Comment:
Strictly speaking, I'm not sure if you should do inherits that late (especially after defining metadata vars) but I don't think it's going to break anything here, and it will take a while before I get proper answer from our resident expert, so don't worry about it now ;-).